### ISSUE TITLE
severals windows on pc

### ISSUE THREAD

**Issue by toto6976** [Genymobile/scrcpy#5490]
Hi,

Is it possible to have two windows open on PC for two separate devices?

**Comment by rom1v** [comment]
Yes. https://github.com/Genymobile/scrcpy/blob/master/doc/connection.md#selection

**Comment by toto6976** [comment]
Hi,

My case does not correspond to any of those cited in the file: the two smartphones are connected via TCPIP (without direct USB connection).

**Comment by rom1v** [comment]
Whatever, this page explains how to select a device when several are connected.

Run one instance by selecting one device and another instance for the other device.
